src/utils/client.ts

```diff
@@ -1036,6 +1036,21 @@ export default class Client {
       } else throw Error(parseError(await response.json()));
     }
   }
+
+  async sendEmail(templateId: string) {
```

Comment:
could we pull all of the integration actions into a separate file? maybe it can be a base class inherited by the client.ts class here. we've gotten to the point IMO where we have too many functions in one place here.